src/runtime/handle.ts

```diff
@@ -138,6 +138,17 @@ export abstract class Handle {
   protected generateKey(): string {
     return this.idGenerator.newChildId(Id.fromString(this._id), 'key').toString();
   }
+
+  disable(particle?: Particle, storageProxy?: StorageProxy): void {
```

Comment:
Add method documentation explaining what it does (I realize we have limited documentation on the class to date, but IMO we should start enhancing things).

Will we actually need to allow a null `storageProxy` in some cases?

Is now a good time to add a unit test for this?

Reply:
Changed this into requiring a Store parameter and check if the Store is an instance of StorageProxy before calling deregister() function owned by StorageProxy.